v2/cmd/driver/main.go

```diff
@@ -134,28 +139,40 @@ func drive() (err error) {
 		return err
 	}
 	if execution.ID != 0 {
+		glog.Infof("output execution.ID=%v", execution.ID)
 		if err = writeFile(*executionIDPath, []byte(fmt.Sprint(execution.ID))); err != nil {
 			return fmt.Errorf("failed to write execution ID to file: %w", err)
 		}
 	}
 	if execution.Context != 0 {
+		glog.Infof("output execution.Context=%v", execution.Context)
 		if err = writeFile(*contextIDPath, []byte(fmt.Sprint(execution.Context))); err != nil {
 			return fmt.Errorf("failed to write context ID to file: %w", err)
 		}
 	}
 	if execution.ExecutorInput != nil {
 		marshaler := jsonpb.Marshaler{}
-		executorInputJson, err := marshaler.MarshalToString(execution.ExecutorInput)
+		executorInputJSON, err := marshaler.MarshalToString(execution.ExecutorInput)
 		if err != nil {
 			return fmt.Errorf("failed to marshal ExecutorInput to JSON: %w", err)
 		}
-		if err = writeFile(*executorInputPath, []byte(executorInputJson)); err != nil {
+		glog.Infof("output ExecutorInput:%s\n", prettyPrint(executorInputJSON))
+		if err = writeFile(*executorInputPath, []byte(executorInputJSON)); err != nil {
 			return fmt.Errorf("failed to write ExecutorInput to file: %w", err)
 		}
 	}
 	return nil
 }
 
+func prettyPrint(jsonStr string) string {
+	var prettyJSON bytes.Buffer
+	err := json.Indent(&prettyJSON, []byte(jsonStr), "", "  ")
+	if err != nil {
+		return jsonStr
```

Comment:
Do we need to add some logging here?

Reply:
the function is only used for logging purposes, so I think it's enough returning the original value.